html-forms/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="index.html" method="post">
+      <div><label for="email">Email</label></div>
+      <input id="email" type="text" name="email">
+      </form>
+
+    <form action="index.html" method="post">
+      <div>Why is coding cool?</div>
+      <select name="reason">
+        <option id="select" value="default">Select an option</option>
+        <label>Select an option</label>
+        <option id="interesting" value="interesting">Because it's interesting</option>
+        <label>Because it's interesting</label>
+        <option id="fun" value="fun">Because it's fun</option>
+        <label>Because it's fun</label>
+        <option id="powerful" value="powerful">Because it's powerful</option>
+        <label>Because it's powerful</label>
```

Comment:
`option` elements do not have labels, since they cant be labeled individually. You can remove the `label` elements here.